ichnaea/content/views.py

```diff
@@ -31,16 +31,18 @@
 
 CSP_BASE = "'self'"
 CSP_POLICY = """\
-default-src 'self' *.tiles.mapbox.com;
+default-src 'self';
+connect-src {base} api.mapbox.com;
```

Comment:
The URL for the old tiles was `https://a.tiles.mapbox.com/v4`, this changes to the new URL `api.mapbox.com`. There is [CSP guidance](https://docs.mapbox.com/mapbox-gl-js/overview/#csp-directives) for the GL API (vector tiles), but not the static API. It does mention ``https://*.tiles.mapbox.com``, but my testing didn't show any calls to that URL, or ``https://events.mapbox.com``.

I did see errors about the GA code trying to connect, such as this from Safari:

```
Refused to connect to https://www.google-analytics.com/j/collect?<omitted> because it does not appear in the connect-src directive of the Content Security Policy.
```

I see similar errors in production with Safari, so I suspect Google Analytics has been broken for a long time. I don't think we need to analyze web site visitors for MLS, so I filed [bug 1642422](https://bugzilla.mozilla.org/show_bug.cgi?id=1642422) to remove Google Analytics.